docs/General/Manuals/client-manual.rst

```diff
@@ -1454,6 +1454,32 @@ Unit ID value.
 Mini Map
 --------
 
+On the `Map View`_, in the lower right corner will be a :guilabel:`Mini Map` widget. Figure 43 below shows a
+sample of what the :guilabel:`Mini Map` looks like for a single-player game. Figure 44 shows an example of
+what a :guilabel:`Mini Map` looks like for a LongTurn multi-player game. The big difference between the two
+is the single-player :guilabel:`Turn Done` button is replaced by a countdown timer. On the right hand side of
+the :guilabel:`Mini Map` are some zoom buttons similar to what you would find in the `View Menu`_. Inside of
+the :guilabel:`Mini Map` you will see a square. This is the portion of the whole `Map View`_ that is displyed
```

Comment:
"...you will see ~~a square~~ *an area with a white border*."

It's a rectangle or a rhombus, but using a generic "quadrilateral" seems a little too verbose